### PR TITLE
Feature/22 domain structure of money wallet

### PR DESCRIPTION
**Pull request by Iwona007**
refactor: code review improvement for branch  DawidStuzynski:feature/domain-structure-of-money-wallet
style: delete empty lines

### CODE REVIEW

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Please hold off with any changes. I need to make sure how to version the classes, i.e .:
_M.M.S.P_
or
_M.M.P_

Also, whether class numbering in the format is needed
_version_
or
_version + modification date_

**Review by mirekgab — APPROVED** [review]

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java [hunk 1]
```diff
@@ -8,6 +8,9 @@
 import pl.byczazagroda.trackexpensesappbackend.exception.AppException;
 import pl.byczazagroda.trackexpensesappbackend.exception.AppRuntimeException;
 
+/**
+ * GlobalExceptionHandlerController exception handler for all application.
```
Comment (dawciobiel) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java:12]: Suggested content:
" GlobalExceptionHandlerController exception handler for all application **exceptions**."

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * @version 0.2.0
+ */
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class Wallet implements Serializable {
+
+    /**
+     * Class version 0.1.0.  SerialVersionUID needs to be updated with any change
```
Comment (dawciobiel) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java:31]: Dot in the end of the sentence. Please.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppRuntimeException.java [hunk 3]
```diff
@@ -1,5 +1,8 @@
 package pl.byczazagroda.trackexpensesappbackend.exception;
 
+/**
+ * AppRuntimeException business exception {@code RuntimeException}
```
Comment (dawciobiel) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppRuntimeException.java:4]: Dot in the end of the sentence. Please.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppException.java [hunk 4]
```diff
@@ -1,5 +1,8 @@
 package pl.byczazagroda.trackexpensesappbackend.exception;
 
+/**
+ * AppException business exception {@code Exception}
```
Comment (dawciobiel) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppException.java:4]: Dot in the end of the sentence. Please.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/WalletDTO.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import java.time.Instant;
+
+/**
+ *
```
[the remark this answers is not in the document]
Comment (dawciobiel) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/WalletDTO.java:6]: Please add at least: "Wallet Data Transfer Object class" as first line of the javadoc on the class.

**Review by dawciobiel — CHANGES_REQUESTED** [review]

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/WalletDTO.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/WalletDTO.java:7]: I added a description

**Review by dawciobiel — APPROVED** [review]

**Review by mirekgab — APPROVED** [review]

**Comment by mirekgab** [comment]
> Please hold off with any changes. I need to make sure how to version the classes, i.e .: _M.M.S.P_ or _M.M.P_
> 
The classes will be versioned according to the scheme M.M.P (Major.Minor.Patch). The number will match the milestone.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
+package pl.byczazagroda.trackexpensesappbackend.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.byczazagroda.trackexpensesappbackend.service.WalletService;
+
+/**
+ * Controller for Wallet application
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java]: Corrected.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java [hunk 7]
```diff
@@ -8,6 +8,9 @@
 import pl.byczazagroda.trackexpensesappbackend.exception.AppException;
 import pl.byczazagroda.trackexpensesappbackend.exception.AppRuntimeException;
 
+/**
+ * GlobalExceptionHandlerController exception handler for all application
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java]: Corrected.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/TrackExpensesAppBackendApplication.java [hunk 8]
```diff
@@ -9,5 +9,4 @@ public class TrackExpensesAppBackendApplication {
     public static void main(String[] args) {
         SpringApplication.run(TrackExpensesAppBackendApplication.class, args);
     }
-
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/TrackExpensesAppBackendApplication.java:12]: The notice is correct but to save time the change is accepted.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java [hunk 9]
```diff
@@ -17,5 +18,4 @@ public class HelloWorldController {
     public String sayHello() {
         return helloWorldService.sayHello();
     }
-
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java:20]: The notice is correct but to save time the change is accepted.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java (hunk 1 above)
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java]: Corrected.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/config/ApplicationConfig.java [hunk 10]
```diff
@@ -26,5 +26,4 @@ public void customizeObjectMapper() {
         objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         objectMapper.configure(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES, true);
     }
-
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/config/ApplicationConfig.java:29]: The notice is correct but to save time the change is accepted.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppException.java (hunk 4 above)
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppException.java]: Corrected.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppRuntimeException.java (hunk 3 above)
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppRuntimeException.java]: Corrected.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java (hunk 2 above)
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java]: Corrected.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppException.java [hunk 11]
```diff
@@ -1,5 +1,8 @@
 package pl.byczazagroda.trackexpensesappbackend.exception;
 
+/**
+ * AppException business exception {@link Exception}
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppException.java]: The notice is correct but to save time the change is accepted.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java [hunk 12]
```diff
@@ -6,6 +6,7 @@
 import org.springframework.web.bind.annotation.RestController;
 import pl.byczazagroda.trackexpensesappbackend.service.HelloWorldService;
 
+//todo this Mock Class. It should be deleted before release 1.0.0
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java:9]: The notice is correct but to save time the change is accepted.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/repository/WalletRepository.java [hunk 13]
```diff
@@ -0,0 +1,11 @@
+package pl.byczazagroda.trackexpensesappbackend.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
+
+/**
+ * Repository for Wallet Entity
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/repository/WalletRepository.java]: The notice is correct but to save time the change is accepted.

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppRuntimeException.java [hunk 14]
```diff
@@ -1,5 +1,8 @@
 package pl.byczazagroda.trackexpensesappbackend.exception;
 
+/**
+ * AppRuntimeException business exception {@link RuntimeException}
```
[the remark this answers is not in the document]
Comment (mirekgab) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/AppRuntimeException.java]: The notice is correct but to save time the change is accepted.

**Comment by Iwona007** [comment]
create domain structure of money wallet

**Pull request merged**